st2common/st2common/services/datastore.py

```diff
@@ -185,7 +188,8 @@ def delete_value(self, name, local=True, scope=SYSTEM_SCOPE):
         try:
             params = {'scope': scope}
             client.keys.delete(instance=instance, params=params)
-        except Exception:
+        except Exception, e:
+            self._logger.warning('Exception deleting value from datastore (name=%s): %s', name, e)
```

Comment:
Perhaps a good idea to use log.exception so the actual exception / stacktrace is also logged.